services/ui-src/src/components/reports/ModalOverlayReportPage.tsx

```diff
@@ -41,20 +48,52 @@ export const ModalOverlayReportPage = ({ route }: Props) => {
     addEditEntityModalOnCloseHandler();
   };
 
+  // delete modal disclosure and methods
+  const {
+    isOpen: deleteEntityModalIsOpen,
+    onOpen: deleteEntityModalOnOpenHandler,
+    onClose: deleteEntityModalOnCloseHandler,
+  } = useDisclosure();
+
+  const openDeleteEntityModal = (entity: EntityShape) => {
+    setSelectedEntity(entity);
+    deleteEntityModalOnOpenHandler();
+  };
+
+  const closeDeleteEntityModal = () => {
+    setSelectedEntity(undefined);
+    deleteEntityModalOnCloseHandler();
+  };
+
   return (
     <Box data-testid="modal-overlay-report-page">
       {verbiage.intro && (
         <ReportPageIntro
           text={verbiage.intro}
-          accordion={accordionVerbiage.formIntro}
+          accordion={accordionVerbiage.MLR.formIntro}
```

Comment:
req: I'd prefer we didn't hardcode this to MLR. can we make use of the report type to determine which formIntro we should use?